src/github/pullRequestModel.ts

```diff
@@ -1201,6 +1274,10 @@ export class PullRequestModel extends IssueModel<PullRequest> implements IPullRe
 			files = data.files as IRawFileChange[];
 		}
 
+		if (oldHasChangesSinceReview !== this.hasChangesSinceLastReview) {
+			this._onDidChangeChangesSinceReview.fire({ afterActivation: oldHasChangesSinceReview === undefined });
```

Comment:
This is done for the case when the user reopens VSCode and the last time he had the diffTab open in the changesSinceReview mode. In that case it will reopen the diff in the default state (show all Diffs)